iwyu_tool.py

```diff
@@ -197,6 +214,30 @@ def slice_compilation_db(compilation_db, selection):
 
     return new_db
 
+def process_invocations(invocations, verbose, formatter, jobs):
+    if jobs == 1:
```

Comment:
This special case seems unnecessary to me -- the algorithm below already handles that just fine.

Reply:
It's not, it actually does something different. Instead of busy-waiting, it waits on the process to complete directly. Also it avoids all the queue shuffling and state checking required below.